frontend/src/metabase/nav/containers/Navbar.unit.spec.tsx

```diff
@@ -25,25 +27,26 @@ type SetupOpts = {
   route?: string;
   pathname?: string;
   user?: User | null;
+  embedOptions?: Record<string, string | boolean>;
 };
 
 async function setup({
   isOpen = true,
   pathname = "/",
   route = pathname,
   user = createMockUser(),
+  embedOptions = {},
 }: SetupOpts = {}) {
   const hasContentToFetch = !!user;
   const isAdminApp = pathname.startsWith("/admin");
 
-  if (hasContentToFetch) {
```

Comment:
I had to remove this conditional for the existing tests because it caused subsequent tests to falsely fail if these endpoints weren't mocked.